src/api/public/apidocs-new/paths/search_channel.yaml

```diff
@@ -0,0 +1,83 @@
+get:
+  summary: List channel objects that match a XPath condition.
+  description: |
+    Return a collection of channel objects with release informations that match a XPath condition.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: match
+      schema:
+        type: string
+      required: yes
+      description: |
+        Expression based in XPath.
+        Providing a value of `*` will return all channels.
+        Available predicates are:
```

Comment:
```suggestion
        Expression based in XPath.

        Providing a value of `*` will return all channels.

        Available predicates are: 
```
Without adding a blank line in between all text is rendered in the same line. I find it more clear to split the text in different paragraphs.

![Screenshot from 2022-08-25 16-53-14](https://user-images.githubusercontent.com/24919/186698437-8bfb2d87-abdb-4903-a423-363110a38959.png)